webapp/locales/de.json

```diff
@@ -82,8 +82,14 @@
     "download": {
       "name": "Daten herunterladen"
     },
-    "delete": {
-      "name": "Konto löschen"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-13T17:48:13Z" title="Thursday, June 13th 2019, 7:48:13 pm +02:00">Jun 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -25,16 +25,7 @@
     "shouted": "Empfohlen",
     "commented": "Kommentiert",
     "userAnonym": "Anonymus",
-    "socialMedia": "Wo sonst finde ich",
-    "network": {
```


@mattwr18 why did you delete the translations? :thinking:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-13T18:18:44Z" title="Thursday, June 13th 2019, 8:18:44 pm +02:00">Jun 13, 2019</time>_

---


I didn't! well, I must have worked through a merge conflict incorrectly... I'll add it back
weird thing is that trying to pull in the latest from master says it's up to date :woman_shrugging: